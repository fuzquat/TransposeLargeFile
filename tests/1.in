1a	1b	1c
2a	2b	2c

