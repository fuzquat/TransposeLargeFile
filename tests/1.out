1a 2a
1b 2b
1c 2c

